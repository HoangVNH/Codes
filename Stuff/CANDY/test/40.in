10
97
VXDXXXVVVDXVDDVVXXVDXVDVXXXDXDVVVVDDXXXXXDVVXDVDDXDVXDVDXVVDXVXXVVXVVXDVXDXXXVVDDVVVVDVXXXVVXXXXD
50
XVVXDVDDVDXDDVDXXDVDXXVXVVVVVDDDXVVVXXDDDDDVXXVDDV
56
VXDVVDDVXVXDDVDDDDVXDXDDVXVVXXVVVXVVDDDXVXDDDVVXXXXDDVVD
54
DVDXVDVVVDXVDXXXDVVVVXXXXDXVDDVVXVXVDVVVVVXXVVDDDDDXXV
55
XVDVVVXDVDXXXXVVVXVDVXDVVDVXXDXDDXXXXDVXXVVXVVXVDDDVDDX
90
XVVXDVDDDXDDVDXDVDVDVDXDVDVVDXVXVVVDXVDVDDXVDDDDVDDVDVDXXDVVDVXXDDXVXDDDDXVXXDVDXDDVDXDDDV
62
DXDVDDDDXDVVDVDXDVVVVXVXDXDVDDDDXDVDXXVDXVVVVDVDVXXXVDDDDXXXVD
56
VVXDDXXDXXDVXVVDXXVXVDDDDDXDDXDDXVVVVXDVVVDXXVVVVVVVDXVV
92
DVXXVVDVDVVDVDVVVDXXDXXVXXDXXXDXXDVDDXXVVVXVVXVVDVXXXVXXVXDDXXXXVDVXVXVXXDXVXXVXVDDDDDDVVXVD
70
VVDVDVXXXXVDXVVXDVDDDVVXDXXVXDDVVDVDVVVDXVDVDXVVXVDXVXDXVDXVDDDDXXDXDD
