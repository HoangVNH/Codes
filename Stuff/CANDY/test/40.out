96
49
55
53
54
88
60
55
90
69
